patchwork-events-entity/src/main/java/net/patchworkmc/mixin/event/entity/MixinEntity.java

```diff
@@ -52,4 +56,14 @@ public void hookConstructor(EntityType<?> type, World world, CallbackInfo ci) {
 
 		EntityEvents.onEntityConstruct(entity);
 	}
+
+	@Inject(method = "startRiding(Lnet/minecraft/entity/Entity;Z)Z", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/Entity;canStartRiding(Lnet/minecraft/entity/Entity;)Z"))
```

Comment:
comment: will need to double-check this injection before merge and make sure this can't accidentally not get fired
If you go by the source it looks correct, but bytecode control flow is *really weird* and I want to be more sure